packages/app/src/hooks/endpoint.ts

```diff
@@ -922,3 +923,28 @@ export const useEndpoint = (): UseEndpointReturn => {
   );
   return ret;
 };
+
+export const useEndpointGroupToggle = (id: string) => {
+  const [endpointGroups, setEndpointGroups] = useEndpointGroupToggleState();
+
+  const isOpenToggle =
+    endpointGroups.find((group) => group.id === id)?.isOpenToggle ?? false;
+
+  const setIsOpenToggle = () => {
```

Comment:
Functions that start with "set" are typically considered as setters for private properties. Nonetheless, the absence of arguments in such a function could make its behavior unpredictable.